examples/md-flexible/src/Simulation.cpp

```diff
@@ -372,6 +465,106 @@ void Simulation::calculateGlobalForces(const std::array<double, 3> &globalForce)
+    std::cout << timerToString("  Initialization             ", initialization, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("  Simulate                   ", simulate, maximumNumberOfDigits, total);
+    std::cout << timerToString("  PositionUpdate             ", positionUpdate, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("  Boundaries:                ", haloParticleExchange + migratingParticleExchange,
+                               maximumNumberOfDigits, simulate);
+    std::cout << timerToString("    HaloParticleExchange     ", haloParticleExchange, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("    MigratingParticleExchange", migratingParticleExchange, maximumNumberOfDigits,
+                               simulate);
+    std::cout << timerToString("  ForceUpdateTotal           ", forceUpdateTotal, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("    ForceUpdatePairwise      ", forceUpdatePairwise, maximumNumberOfDigits,
+                               forceUpdateTotal);
+    std::cout << timerToString("    ForceUdpateGlobalForces  ", forceUpdateGlobalForces, maximumNumberOfDigits,
+                               forceUpdateTotal);
+    std::cout << timerToString("    ForceUpdateTuning        ", forceUpdateTuning, maximumNumberOfDigits,
+                               forceUpdateTotal);
+    std::cout << timerToString("    ForceUpdateNonTuninng    ", forceUpdateNonTuning, maximumNumberOfDigits,
+                               forceUpdateTotal);
+    std::cout << timerToString("  VelocityUpdate             ", velocityUpdate, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("  Thermostat                 ", thermostat, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("  Vtk                        ", vtk, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("One iteration                ", total / _iteration, maximumNumberOfDigits, simulate);
+
+    const long wallClockTime = _timers.total.getTotalTime();
+    std::cout << timerToString("Total wall-clock time        ", wallClockTime, std::to_string(wallClockTime).length(),
+                               wallClockTime);
+    std::cout << std::endl;
+
+    std::cout << "Tuning iterations             : " << _numTuningIterations << " / " << _iteration << " = "
+              << ((double)_numTuningIterations / _iteration * 100) << "%" << std::endl;
+
+    auto mfups = _autoPasContainer->getNumberOfParticles(autopas::IteratorBehavior::owned) * _iteration * 1e-6 /
+                 (forceUpdateTotal * 1e-9);  // 1e-9 for ns to s, 1e-6 for M in MFUP
+    std::cout << "MFUPs/sec                     : " << mfups << std::endl;
+
+    if (_configuration.dontMeasureFlops.value) {
+      autopas::FlopCounterFunctor<ParticleType> flopCounterFunctor(_autoPasContainer->getCutoff());
+      _autoPasContainer->iteratePairwise(&flopCounterFunctor);
+
+      size_t flopsPerKernelCall;
```

Comment:
you miss to initialize this variable. Here the old code:
```suggestion
      size_t flopsPerKernelCall;
        switch (_config->functorOption.value) {
    case MDFlexConfig::FunctorOption ::lj12_6: {
      flopsPerKernelCall = autopas::LJFunctor<ParticleType, _shifting, _mixing>::getNumFlopsPerKernelCall();
      break;
    }
    case MDFlexConfig::FunctorOption ::lj12_6_Globals: {
      flopsPerKernelCall = autopas::LJFunctor<ParticleType, _shifting, _mixing, autopas::FunctorN3Modes::Both,
                                              /* globals */ true>::getNumFlopsPerKernelCall();
      break;
    }
    case MDFlexConfig::FunctorOption ::lj12_6_AVX: {
      flopsPerKernelCall = autopas::LJFunctorAVX<ParticleType, _shifting, _mixing>::getNumFlopsPerKernelCall();
      break;
    }
    default:
      throw std::runtime_error("Invalid Functor choice");
  }
```